test/e2e/user_workload_monitoring_test.go

```diff
@@ -681,6 +703,123 @@ func assertTenancyForMetrics(t *testing.T) {
 	if err != nil {
 		t.Fatalf("the account has access to the rules endpoint of Thanos querier: %v", err)
 	}
+
+	for _, tc := range []struct {
+		// NOTE: role verbs need to factor in the permissions metrics-server requires for the respective use-case.
```

Comment:
Not sure I follow the meaning here? Is this still up-to-date or am I missing something? If the later may a bit more context might be good?

Reply:
Ah, this seems to be an artefact from earlier patches on this PR, not relevant now. Dropped.